drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -141,7 +142,7 @@ local function migrate_bridge(driver, device)
     }
 
     device:try_update_metadata(new_metadata)
-    log.trace("Bridge Migrated, re-adding")
+    log.trace_with({ hub_logs = true }, "Bridge Migrated, re-adding")
```

Comment:
Seems like this could be info, but perhaps other device init logging indicates successful migration as well.